R/shiny-recorder.R

```diff
@@ -455,21 +455,49 @@ RecordingSession <- R6::R6Class("RecordingSession",
 )
 
 #' Record a Session for Load Test
-#' @details This function creates a reverse proxy listening at \code{host:port}
-#'   that stands in front of \code{target_app_url}. As the user interacts with
-#'   the app, the actions and responses are recorded in a file that can be
-#'   replayed later. By default, a web browser is opened automatically so that
-#'   you can begin recording immediately, without manually starting a browser
-#'   and navigating to \code{host:port}.
-#' @param target_app_url The URL of the deployed application
+#'
+#' This function creates a [reverse
+#' proxy](https://en.wikipedia.org/wiki/Reverse_proxy) at `http://host:port`
+#' (http://127.0.0.1:8600 by default) that intercepts and records activity
+#' between your web browser and the Shiny application at `target_app_url`.
+#'
+#' By default, after creating the reverse proxy, a web browser is opened
+#' automatically. As you interact with the application in the web browser,
+#' activity is written to the `output_file` (`recording.log` by default).
+#'
+#' To shut down the reverse proxy and complete the recording, close the web
+#' browser tab or window.
+#'
+#' Recordings are used as input to the `shinycannon` command-line
+#' load-generation tool which can be obtained from the [shinyloadtest
```

Comment:
maybe add backticks on `shinyloadtest`

Reply:
Hm, I agree, but when I do I don't see a difference. Gonna let it ride.